### ISSUE TITLE
some issues with snow and poisoning, accessibility options and more

### ISSUE THREAD

**Issue by queermeet-dev** [PLAYERUNKNOWN-Productions/go-wayback-user-support#403]
when drinking snow you are poisoned, that makes absolutely no sense and then the poisoning is lasting wayyyyy too long

Then: you can take snow with the bottle but not with the pot so you cannot boil the snow to drink it

the map should be a little more interactive, not everyone can read and understand normal maps and knows how to navigate, you should build in accessibility options for people with disabilities

you should give better instructions at the beginning, how to barricade windows properly, how to survive and all that stuff, make it nore beginner friendly

**Comment by DWaterMalone** [comment]
Thanks for the feedback, and apologies for the delay in our response. We're considering next steps on how we could address your points!
